discovery-infra/bm_inventory_api.py

```diff
@@ -132,5 +132,6 @@ def create_client(inventory_url=None, wait_for_url=True):
         i_url = utils.get_service_url("bm-inventory")
     log.info("Inventory url %s", i_url)
     client = InventoryClient(inventory_url=i_url)
-    client.wait_for_api_readiness()
+    if not wait_for_url:
```

Comment:
Doesn't the condition suppose to be the opposite? My logic sees a "wait" boolean -> so wait.

In case not - I suggest to change the names of the variable/function to be more accurate.

Reply:
next time